### PR TITLE
The russian quoteHeadersRegex should not check for "wrote:"

### PR DESCRIPTION
**Pull request by SL-Gundam**
The Ukrainian regex checks for "wrote:" causing it to accept "Somebody wrote:" as a header line for a reply. Believe wrote should already be covered in the other regex's.

I adjusted the Ukrainian test case to have wrote in the text so that it would fail with the old regex, not with the new one.

Since this is very specific i had to modify the case and add a tiny cleanup code so that travis would properly fail or pass the adjustments

### CODE REVIEW

File: tests/EmailReplyParser/Tests/Parser/EmailParserTest.php [hunk 1]
```diff
@@ -161,6 +161,7 @@ public function testEmailUkrainian()
     {
         $email     = $this->parser->parse($this->getFixtures('email_23.txt'));
         $fragments = $email->getFragments();
+        $fragments[0] = str_replace("\nSomebody wrote:\n", '', $fragments[0]);
```
Comment (willdurand) [inline tests/EmailReplyParser/Tests/Parser/EmailParserTest.php]: I dont quite understand this change in the tests :/ Could you create another test case with a different fixtures file? I think I understand the bug you've fixed, but why is this necessary?

Comment (SL-Gundam) [reply tests/EmailReplyParser/Tests/Parser/EmailParserTest.php]: If "Somebody wrote:" is in the body of an email it will be seen as a quote header. I needed this before the actual quote header because the test cases only verify the first fragment.

The old regex would only find half of the original text needed for a pass. The new regex would find the proper quote header and ignore "Somebody wrote:".

Since the text all test cases verify themselves with does not contain "Somebody wrote:", it needs to be removed so that proper verification can be done (hence the line your talking about).

The current Ukrainian test case does not warrant "wrote:" in the regex either.

If i build a new test case, that case would pass with the new regex. Only the old regex would fail.
I personally don't see how we could build a proper test case that would be understandable in the future for why it was ever created.
If it were up to me i would not add a test case for this since the default text being verified against does not contain this flaw.

If you still want me to create a test case, i will do it but please answer these questions
Do you want the test case to fail or pass with the old regex?
Do you want the test case to fail or pass with the new regex?
I'm asking because Travis will only be able to verify it against the new regex

Comment (willdurand) [reply tests/EmailReplyParser/Tests/Parser/EmailParserTest.php]: I am sorry but I still don't quite understand.

You've modified the regex, fine, but if it breaks the test suite, it introduces a regression. As far as I know, the test suite is quite OK since no one really complained 😋 

If you've found a bug, then I'd like to have it revealed by a new test case. If the problem is actually covered by all tests, hence everything red, then it is a regression and I'd like the big picture to understand how to treat it: new major version, bug fix, or wont fix.

Then again, sorry, my brain does not work well it seems.

Comment (SL-Gundam) [reply tests/EmailReplyParser/Tests/Parser/EmailParserTest.php]: The current test suite is not able to check for this since it currently does not contain a line "anything here wrote:" or "Something wrote:" in the text here: https://github.com/willdurand/EmailReplyParser/blob/master/tests/EmailReplyParser/Tests/Parser/EmailParserTest.php#L10
Any line ending in "wrote:" with a space and a single letter in front of it would be valid for the Ukrainian regex

Its not really a bug but the current Ukrainian regex also finds the english reply header because it checks for "wrote:" and could make a mistake since it only checks for that word and does not validate any further like the actual english regex does

I will make a new test and separate the commits
The first commit will modify the test suites. This will make the Ukrainian test fail.
The second commit will fix the regex so that it will be pass again
